source/_integrations/opower.markdown

```diff
@@ -91,6 +91,10 @@ For gas:
 
 Note the unit for gas is CCF (centum cubic feet). 1 CCF is one hundred cubic feet which is equivalent to 1 therm.
 
+Depending on the utility and your setup, there may be multiple meters for a single utility account.
+If this is the case, your sensors will use a UUID to distinguish the data streams within that single account.
```

Comment:
UUID is a rather technical term, can we avoid that?

Reply:
Removed it and replaced to explain that the id will not be tied back to anything the energy provider provides.